cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -66,8 +66,9 @@ if [[ -n "${DB_ADMIN_PASSWORD_URI}" ]]; then
 else
   readonly DB_ADMIN_PASSWORD=''
 fi
+
 if [[ -z ${DB_ADMIN_PASSWORD} ]]; then
-  readonly DB_ADMIN_PASSWORD_PARAMETER=''
+  readonly DB_ADMIN_PASSWORD_PARAMETER='-p'
```

Comment:
I'm a little confused about this change. I check MySQL CLI doc [1], if the flag is not set, the default is to send no password.

Also can you replace `-p<password>` with `--password=<password>` in this script, the latter is less error-prone.

https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_password

Reply:
Quotes added and changed -p to a safer --password=